.github/workflows/ruby-core.yml

```diff
@@ -50,8 +50,13 @@ jobs:
         with:
           path: ruby/rdoc
       - name: Build RDoc locally
+        # The `build` task provided by `require 'bundler/gem_tasks'` seems to have a bug
+        # as described in https://github.com/rubygems/rubygems/issues/8477
+        # The manual `gem build` and `gem install` are used to work around this issue.
         run: |
           bundle install
+          gem build rdoc.gemspec
+          gem install rdoc-*.gem
```

Comment:
Do we need `bundle exec ...` here?

Reply:
I don’t think so…? I don’t usually run `gem` with bundle as it usually doesn’t need any of the project’s dependencies. Is it a recommended practice?

Reply:
Ah, sorry. You're right.

Our `.gemspec` doesn't use external libraries. So we don't need `bundle exec` here.